src/Serialization/Hash.js

```diff
@@ -0,0 +1,61 @@
+const CardanoWasm = require("@emurgo/cardano-serialization-lib-nodejs");
+
+const hashFromImpl = hashClassFrom => maybe => input => {
+    const ret = null;
+    try {
+        ret = hashClassFrom(input);
+    }
+    catch (e) {
+        console.log(e);
+    }
+    if (ret == null) {
+        return maybe.nothing;
+    }
+    return maybe.just(ret);
```

Comment:
```suggestion
    try {
        return maybe.just(hashClassFrom(input));
    }
    catch (e) {
        console.log(e);
        return maybe.nothing;
    }
```

I'd remove logging as well.